auth/authn.go

```diff
@@ -46,14 +44,24 @@ var (
 
 	Authority *authority.Authority
 
-	authEnabled bool
-	signingKey  []byte
-	tokenStore  TokenStore
+	authEnabled      bool
+	signingKeyBase64 []byte
```

Comment:
I am just curious why we need to use both  base64 encoded and base32 encoded key.